app.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const {} = require("./controllers/articles.controller");
+const {} = require("./controllers/comments.controller");
+const { getAllTopics } = require("./controllers/topics.controller");
+const {} = require("./controllers/users.controllers");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
You have no use for an express.json in this ticket.

I would delete this and only put it in when you come across a ticket that needs it. From a dev point of view imagine working in a team project and putting in code that's unrelated to your work and potentially someone else will be adding because it's actually part of their ticket. It would make no sense and potentially lead to conflicts